tests/Behat/Page/Admin/PriceHistory/IndexPage.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Page\Admin\PriceHistory;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPage as BaseIndexPage;
+
+class IndexPage extends BaseIndexPage implements IndexPageInterface
+{
+    public function isThereLogEntryWithPriceAndOriginalPrice(string $price, string $originalPrice): bool {
+        $availablePrices = $this->getColumnFields('price');
+        $availableOriginalPrices = $this->getColumnFields('originalPrice');
+
+        foreach ($availablePrices as $key => $value) {
+            if (
+                $this->formatIntoPlainNumber($availablePrices[$key]) === $price &&
+                $this->formatIntoPlainNumber($availableOriginalPrices[$key]) === $originalPrice
+            ) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    public function formatIntoPlainNumber(string $price): string
+    {
+        return str_replace(['.', '$', ','], '', $price);
```

Comment:
I would be in favour of not removing the currency
```suggestion
        return str_replace(['.', ','], '', $price);
```
and preferably to check the exact value without using this formatter